ClientRuntimes/CSharp/Microsoft.Rest.ClientRuntime/project.json

```diff
@@ -24,6 +24,9 @@
         "System.Net.Http": "",
         "System.Net.Http.WebRequest": "",
         "System.Xml": ""
+      },
+      "dependencies": {
+        "Newtonsoft.Json": "6.0.8"
```

Comment:
IMO, the dependency version should be the minimum, we can move to a higher one but should come with a reason that the higher one provide what we need. This way, the library can have the most audience. For autorest, we have to use 7.0 because of the dnx support, out of dnx scenario, the 6.0.x is enough.